doc/manpages/options.asciidoc

```diff
@@ -5,6 +5,37 @@ NAME
 ----
 options - a
 
+Description
+-----------
+
+Options are used for user configuration.
+You can set the builtin ones or declare your owns.
+
+The following commands are available to interact with them:
+
+----------------------------------------------
+declare-option [-hidden] <type> <name> [value]
+set-option [-add] <scope> <name> <value>
+update-option <scope> <name>
+unset-option <scope> <name>
+----------------------------------------------
+
+Refer to the 'scopes' documentation page for more information about
+scopes.
+
+To get the current value of an option:
```

Comment:
Yes, I see what you mean.
It puts more burden on our shoulders as doc maintainers with the benefits for kakoune newcomers to have the info right where they need it.

What mechanism could be used to let the user easily jump from one topic to another like hyperlink? vim's `:help` tag system is very nice in this regard.